instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import './CommentSection.css';
+
+const CommentSection = props => {
+    return (
+        <div className='comment-section-container'>
+            {props.post.comments.map(comment => (
+                <div className='comment-wrapper'>
```

Comment:
This is where you can map into a component built out in a separate, imported file. It would add some clarity, but good job breaking it down.